rllib/agents/a3c/a3c_torch_policy.py

```diff
@@ -9,8 +11,10 @@
 
 
 def actor_critic_loss(policy, model, dist_class, train_batch):
-    logits, _ = model.from_batch(train_batch)
-    values = model.value_function()
+    with policy._model_lock:
```

Comment:
Instead of inserting the model lock inside the policy loss (which is kind of violating the abstraction since the loss shouldn't know about the lock), why not add the locking to TorchPolicy itself?

I.e., add a global lock used by all methods of torch policy.